src/win/fs.c

```diff
@@ -115,8 +115,8 @@
 
 #define TIME_T_TO_FILETIME(time, filetime_ptr)                              \
   do {                                                                      \
-    uint64_t bigtime = ((uint64_t) ((time) * 10000000ULL)) +                \
-                                  116444736000000000ULL;                    \
+    uint64_t bigtime = ((uint64_t) ((time) * (uint64_t) 10000 * 1000U)) +   \
+                       (uint64_t) 116444736 * 1000U * 1000U * 1000U;        \
```

Comment:
Maybe define `MILLION` and `BILLION` macros and use those. Just a suggestion though, this isn't worse than what it replaces.